oojs.js

```diff
@@ -10,3 +10,31 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+
+class Playlist {
+  constructor(title, songs) {
+    this.title = title;
+    this.songs = [];
+  }
+
+  this.favorites = 0;
+
+  addSong () {
+    if (this.title === true) {
+      this.songs = this.title
+      return true
+    }
+    else {
+      return false
+    }
+  }
+}
```

Comment:
The intention here is that you would push a new song into the `songs` array. So `addSong` needs an argument, and somewhere in your method you would need to do `this.songs.push(newSong)`.